docs/DEVELOPER-osx.md

```diff
@@ -1,10 +1,8 @@
-### Prerequisites
-  * docker and compose
-    * enable Kubernetes in the Docker Desktop settings
-
-### Setup details
-  * install [Taskfile](https://taskfile.dev/installation/)
```

Comment:
I'd like to keep this in here

Reply:
Ok yeah I forgot that we linked to the install instructions on the main readme, so this doesn't need to include that.